demo-linker.css

```diff
@@ -1,12 +1,12 @@
 [data-demo-linker-item] {
 	cursor: pointer;
-	outline: 1px solid mediumvioletred;
+	outline: 1px solid black;
```

Comment:
I do still think it's good to have a default color that stands out when not using rainbow. Not sure if changes to the CSS file need to be in this PR.